gauntlet/packages/gauntlet-serum-multisig/src/commands/multisig.ts

```diff
@@ -0,0 +1,180 @@
+import { Result } from '@chainlink/gauntlet-core'
+import { TransactionResponse, SolanaCommand, RawTransaction } from '@chainlink/gauntlet-solana'
+
+import { logger, BN } from '@chainlink/gauntlet-core/dist/utils'
+import { PublicKey, SYSVAR_RENT_PUBKEY, Keypair } from '@solana/web3.js'
+import { CONTRACT_LIST, getContract } from '@chainlink/gauntlet-solana-contracts'
+import { Idl, Program } from '@project-serum/anchor'
+
+export const wrapCommand = (command) => {
+  return class Multisig extends SolanaCommand {
+    command: SolanaCommand
+    program: Program<Idl>
+    proposal: PublicKey
+    multisigAddress: PublicKey
+    multisigSigner: PublicKey
+    owners: [PublicKey]
+    threshold: number
+
+    static id = `${command.id}`
+
+    constructor(flags, args) {
+      super(flags, args)
+      logger.info(`Running ${command.id} command using Serum Multisig`)
+
+      this.command = new command(flags, args)
+      this.command.invokeMiddlewares(this.command, this.command.middlewares)
+    }
+
+    execute = async () => {
+      this.require(process.env.MULTISIG_ADDRESS != null, 'Please set MULTISIG_ADDRESS env var')
+      this.multisigAddress = new PublicKey(process.env.MULTISIG_ADDRESS || '')
```

Comment:
Move this into the constructor.
```suggestion
      this.require(!!process.env.MULTISIG_ADDRESS, 'Please set MULTISIG_ADDRESS env var')
      this.multisigAddress = new PublicKey(process.env.MULTISIG_ADDRESS)
```